libs/core-wrappers/.browserslistrc

```diff
@@ -0,0 +1,16 @@
+# This file is used by:
+# 1. autoprefixer to adjust CSS to support the below specified browsers
+# 2. babel preset-env to adjust included polyfills
+#
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+#
+# If you need to support different browsers in production, you may tweak the list below.
+
+last 1 Chrome version
+last 1 Firefox version
+last 2 Edge major versions
+last 2 Safari major version
+last 2 iOS major versions
+Firefox ESR
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```

Comment:
**Four `.browserslistrc` Files**
If `/.postcssrc.yml` is loaded from root by `apps/ui-patterns`, then maybe this `.browserlist` can be loaded from root by `apps/ui-patterns` also.

Reply:
Removing `.browserlistrc` (220c325) makes sense. I don't see one in root, but I defaults match UT requirements.

> **Queries**
> 5. If the above methods did not produce a valid result Browserslist will use defaults: `> 0.5%, last 2 versions, Firefox ESR, not dead`. — [browserlist](https://github.com/browserslist/browserslist#queries)

> **Browser requirements**
> The University of Texas at Austin’s websites and Apps (except where a piece of vender technology has other requirements) support the latest browser technology -2 iterations. — [UT](https://brand.utexas.edu/application/web-guidelines/)